app/uk/gov/hmrc/nisp/assets/sass/nirecord.scss

```diff
@@ -76,62 +83,61 @@
 
 
 aside {
-    padding-top: 30px;
-    width:30%;
+  padding-top: 30px;
+  width:30%;
 }
 
 aside h2 {
-    padding-top: 15px;
-    font-weight: bold;
+  padding-top: 15px;
+  font-weight: bold;
 }
 
 .sidebar-border {
+  border-top: 10px solid #005EA5;
+  @media only screen and (max-width:640px){
+      display:none;
 
-    border-top: 10px solid #005EA5;
-    @media only screen and (max-width:640px){
-        display:none;
-
-    }
+  }
 }
 
 .form-label{
-    font-size: 17px;
-    font-weight: bold;
+  font-size: 17px;
+  font-weight: bold;
 }
 
 .full-width {
-    width:100%;
+  width:100%;
 }
 
 .sidebar-mobile {
-    position: absolute;
-    width: 1px;
-    height: 1px;
-    padding: 0;
-    margin: -1px;
-    overflow: hidden;
-    clip: rect(0, 0, 0, 0);
-    border: 0;
-    @media only screen and (max-width:640px){
-
-        position: inherit;
-        width: inherit;
-        height: inherit;
-        padding: inherit;
-        margin: inherit;
-        overflow: inherit;
-        clip: inherit;
-        border: inherit;
-    }
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+  @media only screen and (max-width:640px){
+
+      position: inherit;
```

Comment:
@roundrobin99 I need consistent tabbing for my eyes :eyes:. This pull request has 98 changed lines because of tabbing, if we keep it consistent, this will go away.